### ISSUE TITLE
Failing CI on `master`

### ISSUE THREAD

**Issue by vnepveu** [FuelLabs/fuels-rs#50]
After merging PR #49, CI isn't passing anymore. 
Even though the PR CI was passing before merging. 
Error in CI is: 
```
failures:
---- call_with_structs stdout ----
thread 'call_with_structs' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("Failed to compile core")', fuels-abigen-macro/tests/harness.rs:1028:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by vnepveu** [comment]
Unable to reproduce locally with `cargo test call_with_structs`

**Comment by adlerjohn** [comment]
Original failing run: https://github.com/FuelLabs/fuels-rs/runs/4887398965?check_suite_focus=true

I'm re-running CI to see if it was just a transient issue.

**Comment by vnepveu** [comment]
Apparently this was due to a transient issue. And then to a disconnection because everything took so long. With the merge of https://github.com/FuelLabs/sway/pull/688 that drastically reduces compile time, it should be solved.
A dependency issue fixed in #56 made CI fail but once it's merged it should go back to normal.

**Comment by digorithm** [comment]
This specific CI failure has been captured here https://github.com/FuelLabs/fuels-rs/issues/64. Closing this one.
